kahuna/public/js/directives/gr-image-fade-on-load.js

```diff
@@ -50,13 +50,15 @@ imageFade.directive('grImageFadeOnLoad',
 
             function hide() {
                 element.css({
-                    opacity: 0,
-                    transition: `opacity ${animationDuration}ms ease-out`
+                    opacity: 0
                 });
             }
 
             function reveal() {
-                element.css({opacity: 1});
+                element.css({
+                    opacity: 1,
+                    transition: `opacity ${animationDuration}ms ease-out`
```

Comment:
Do you know if there is any perf problems with applying these to every  element?
The only literature I could find on it [was here](http://www.stubbornella.org/content/2009/03/27/reflows-repaints-css-performance-making-your-javascript-slow/#inline) - which makes sense.